docs/os/windows/windows-vm41.md

```diff
@@ -0,0 +1,203 @@
+Installing a Windows VM
+=======================
+
+Simple Windows install
+----------------------
+
+If you just want something simple and you can live without some features. This works for Windows XP, 7, 10 and 11, and it may work for Windows 8 and 8.1, although this has not been tested.
+
+**Works:**
+- display (1440x900 or 1280x1024 are a nice fit onto FHD hw display)
+- keyboard (incl. correct mapping), pointing device
+- network (emulated Realtek NIC)
+- audio output and input (available even without QWT installation if `qvm-features audio-model` is set as `ich6`)
+
+**Does not work:**
+- copy & paste (the qubes way)
+- copying files into / out of the VM (the qubes way)
+- assigning USB devices (the qubes way via the tray applet)
+- PCI device 5853:0001 (Xen platform device) - no driver
+- all other features/hardware needing special tool/driver support
+
+**Installation procedure:**
+- Have the Windows ISO image (preferrably the 64-bit version) downloaded in some qube.
+- Create a new Qube:
+  - Name: WindowsQube, Color: orange
+  - Standalone Qube not based on a template
+  - Networking: sys-firewall (default)
+  - Launch settings after creation: check
+  - Click "OK".
+- Settings:
+  - Basic:
+    - System storage: 50000+ MB
+  - Advanced:
+    - Include in memory balancing: uncheck
+    - Initial memory: 4096+ MB
+    - Kernel: None
+    - Mode: HVM
+  - Click "Apply".
+  - Click "Boot from CDROM":
+    - "from file in qube":
+      - Select the qube that has the ISO.
+      - Select ISO by clicking "...".
+    - Click "OK" to boot into the windows installer.
+- Windows Installer:
+  - Mostly as usual, but automatic reboots will halt the qube - just restart it again and again until the installation is finished.
+  - Install on first disk.
+  - The Windows license may be read from flash via root in dom0:
+
+    `strings < /sys/firmware/acpi/tables/MSDM`
+
+    Alternatively, you can also try a Windows 7 license key (as of 2018/11
+    they are still accepted for a free upgrade to Windows 10).
+    
+- Afterwards:
+  - In case you switch from `sys-network` to `sys-whonix`, you'll need a static IP network configuration, DHCP won't work for `sys-whonix`.
+  - Use `powercfg -H off` and `disk cleanup` to save some disk space.
+
```

Comment:
split the 2nd line in two different items ? the current phrasing seems to imply that `powercfg` has something to do with disk space, while it actually disables power stuff which is useless for vms, and also fixes the cdrom issue (as in [this issue](https://forum.qubes-os.org/t/win-10-standalone-vm-seems-to-have-no-cdrom-drive/8645/3));

Reply:
O.K.